pkg/network/iperf/iperf.go

```diff
@@ -0,0 +1,35 @@
+package iperf
+
+import (
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zos/pkg/zinit"
+)
+
+const (
+	zinitService = "iperf"
+)
+
+// Ensure creates an iperf zinit service and monitors it
+func Ensure(z *zinit.Client) error {
```

Comment:
The Ensure makes sure that the service exist, right? what if it's already there. 
The right way is u first need to check the status, add it if it does not exist